libs/gi/page-team/src/TeamCharacterSelector.tsx

```diff
@@ -103,9 +114,42 @@ export default function TeamCharacterSelector({
         }
       >
         <CardContent sx={{ display: 'flex', justifyContent: 'center', pb: 0 }}>
-          <Typography variant="h5" display="flex">
-            {team.name}
+          <Typography
+            variant="h5"
+            display="flex"
+            onClick={() => setEditMode(true)}
+          >
+            {teamName}
           </Typography>
+          <Dialog
```

Comment:
I would recommend using `ModalWrapper` for this, even through Dialogs are way nicer to use here, there are some flaws in them that have yet to be addressed: https://github.com/frzyc/genshin-optimizer/issues/1498

Reply:
Sure thing. I'll start making the changes.